README.md

```diff
@@ -1,3 +1,36 @@
 # SnowMan
 
-This is a console Swift application. It plays like Hangman but without the gore.
+Snowman is a Swift console application. The object of the game is to guess the given word or phrase before the snowman "melts". With each incorrect guess, the snowman will lose one of its snowman features until he is but a puddle. 
+
+## Requirements
+
+* [Swift](https://swift.org/) 
+* [Swift Package Manager](https://swift.org/package-manager/)
+
+
+## Running the Application
+
+SnowMan was built using the Apple Platform [Swift Development Snapshot](https://swift.org/download/#latest-development-snapshots) from March 1, 2016 for Apple Swift version 2.2.
+
+To compile the application for use, type 
+```swift
+swift build
+```
+
+Then run the application. 
+
+```swift
+.build/debug/Application
+``` 
+
+
+## Running the Tests
+
+Testing is accomplished using [Swiftest](https://github.com/zachmokahn/Swiftest.git) (version 0.1.8). To run the tests, make sure that the application has been compiled by running 
```

Comment:
Swiftest isn't actually released as version 0.1.8, that's just my fork of it, with a release tagged 0.1.8. Also, you don't need to acknowledge the test runner. I'd remove this whole sentence:
> Testing is accomplished using [Swiftest](https://github.com/zachmokahn/Swiftest.git) (version 0.1.8).